HouseholdManager/appsettings.json

```diff
@@ -6,5 +6,5 @@
     }
   },
   "AllowedHosts": "*",
-  "ConnectionStrings": { "DevConnection": "Server=(local)\\sqlexpress;Database=RocketManagerDb;Trusted_Connection=True;MultipleActiveResultSets=True;" }
```

Comment:
Is this section showing that you're switching from RocketManagerDb to HouseholdManagerDb? Were we still wanting to try to switch to RocketManagerDb for the identity thing Michele was working on? Going to approve this anyway, because that's an easy change later too, plus we will hear updates on Michele's Identity status tonight probably.